sdk/src/types/block/macro.rs

```diff
@@ -2,10 +2,19 @@
 // SPDX-License-Identifier: Apache-2.0
 
 /// Convenience macro to implement and derive base features of identifiers.
-#[macro_export]
+#[macro_export(local_inner_macros)]
 macro_rules! impl_id {
-    ($vis:vis $name:ident, $length:literal, $doc:literal) => {
-        #[doc = $doc]
+    (
+        $(#[$hash_meta:meta])*
```

Comment:
This won't always be a hash though, right ? Can it cause any issue in whatever is being generated later ?

Reply:
no, it's just a variable name